### DISCUSSION TITLE
Shortcut customization to turn pages with mouse buttons

### DISCUSSION THREAD

**Discussion by agileAlligator** [sumatrapdfreader/sumatrapdf#3016] [Q&A]
I have a mouse with mouse button 4 and 5, two extra buttons where the right thumb usually rests.

How should I go about assigning these buttons to go to the next/previous page in a document? Is this even possible?

**Comment by GitHubRulesOK** [comment]
Many programmable mice/joysticks have a means to assign a key sequence to the auxiliary switches, often they come with an enhanced driver. If not there are 3rd party extenders (unsure if your device will work with 4th 5th functions in https://vps2.highrez.co.uk/downloads/X-Mouse%20Button%20Control%20User%20Guide.pdf)

Once you assign keys to such buttons it should also be possible to define those keys as per SumatraPDF commands list.
